exchanges/gateio/gateio_test.go

```diff
@@ -1408,25 +1303,23 @@ func TestCountdownCancelOrders(t *testing.T) {
 func TestCreatePriceTriggeredFuturesOrder(t *testing.T) {
```

Comment:
```suggestion
func TestCreatePriceTriggeredFuturesOrder(t *testing.T) {
	t.Parallel()
	sharedtestvalues.SkipTestIfCredentialsUnset(t, g, canManipulateRealOrders)

	for _, tc := range []struct {
		c currency.Code
		a asset.Item
	}{
		{currency.BTC, asset.CoinMarginedFutures},
		{currency.USDT, asset.USDTMarginedFutures},
	} {
		_, err := g.CreatePriceTriggeredFuturesOrder(t.Context(), tc.c, &FuturesPriceTriggeredOrderParam{
			Initial: FuturesInitial{
				Price:    1234.,
				Size:     2,
				Contract: getPair(t, tc.a),
			},
			Trigger: FuturesTrigger{
				Rule:      1,
				OrderType: "close-short-position",
			},
		})
		assert.NoErrorf(t, err, "CreatePriceTriggeredFuturesOrder should not error for settlement currency: %s, asset: %s", tc.c, tc.a)
	}
}
```

Keeping in spirit with the other cleanups

Reply:
Nice, Thanks 🙏 

Fixed https://github.com/gbjk/gocryptotrader/commit/4805d6ec64